cpp/collections_module/algorithms/collections.hpp

```diff
@@ -0,0 +1,23 @@
+#include <mgp.hpp>
+
+
+void unionAll(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory){
+  mgp::memory=memory;
+  const auto arguments=mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try{
+    mgp::List list1=arguments[0].ValueList();
```

Comment:
`const mgp::List &` to avoid copies

Reply:
If I add that how can I append to list1 and use move on list2 when  the are const?